arch/x86/core/x86_mmu.c

```diff
@@ -1115,10 +1115,10 @@ static pentry_t flags_to_entry(uint32_t flags)
 	return entry_flags;
 }
 
-/* map new region virt..virt+size to phys with provided arch-neutral flags */
-void arch_mem_map(void *virt, uintptr_t phys, size_t size, uint32_t flags)
+/* map new region dest..dest+size to addr with provided arch-neutral flags */
+void arch_mem_map(void *dest, uintptr_t addr, size_t size, uint32_t flags)
 {
-	range_map_unlocked(virt, phys, size, flags_to_entry(flags),
+	range_map_unlocked(dest, addr, size, flags_to_entry(flags),
 			   MASK_ALL, 0);
```

Comment:
This one is actually better with `phys` and `virt`. Could you change the declaration instead?